### PR TITLE
scene.onPointerObservable

### PR DESCRIPTION
**Pull request by BabylonJSGuide**
Hopefully have given it enough love

### CODE REVIEW

**Review by deltakosh — APPROVED** [review]
This is perfect!!!

File: content/How_To/events/Observables.md [hunk 1]
```diff
@@ -4,19 +4,26 @@ PG_TITLE: How to Use Observables
...
+
+* [Playground Example - Simple Custom Observable Example](https://www.babylonjs-playground.com/#6IGFM2)
+An Observable - onXChange- is added to the master sphere. The two minion spheres and the actions they have to undertake form the two Observers which react when a change in the x position of the master is observed.
 
-## Observable
-The implementer uses it to create a property which will trigger all the registered observers. The Generic type T is used to communicate a given data type from the Observable to the Observer.
+## Observable Methods and Properties 
 
-You have the following method/properties:
+The following are available:
```
Comment (deltakosh) [inline content/How_To/events/Observables.md:26]: Oups! We need to mention addOnce which will be executed once and then removed

Comment (BabylonJSGuide) [reply content/How_To/events/Observables.md:26]: Will mention tomorrow. Also I note that in the same section the following states there is a special feature I cannot find mention of it

notifyObservers(): used to notify all the registered Observers (with a little special feature I'll detail further below)

Comment (deltakosh) [reply content/How_To/events/Observables.md:26]: ok thanks!!

**Review by deltakosh — APPROVED** [review]

File: data/statics.json [hunk 2]
```diff
@@ -134,6 +134,11 @@
                     "level": 1,
                     "filename": "Fast_Build"
                 },
+                {
+                    "title": "Interact with a Scene",
+                    "level": 1,
```
Comment (deltakosh) [inline data/statics.json:139]: Filename must be Interactions (capital I) or the build will fail

Comment (BabylonJSGuide) [reply data/statics.json:139]: Done

**Comment by deltakosh** [comment]
Thank you so much!

**Pull request merged**